components/actionbutton/stories/actionbutton.stories.js

```diff
@@ -110,6 +142,45 @@ Quiet.parameters = {
 	},
 };
 
+export const StaticWhiteDefault = TreatmentTemplate.bind({});
+StaticWhiteDefault.tags = ["!dev"];
+StaticWhiteDefault.args = {
+	label: "Edit",
+	iconName: "Edit",
+	staticColor: "white",
+};
+
+StaticWhiteDefault.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const StaticBlackDefault = TreatmentTemplate.bind({});
+StaticBlackDefault.tags = ["!dev"];
+StaticBlackDefault.args = {
+	label: "Edit",
+	iconName: "Edit",
+	staticColor: "black",
+};
+
+StaticBlackDefault.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const Sizing = (args, context) => Sizes({
```

Comment:
Maybe we could use some of the descriptions from [the guidance site](https://spectrum.adobe.com/page/action-button/)? That page has a nice description about each of the stories we have.

I definitely think we should include something about the label, since it's important for accessibility.